simulation.py

```diff
@@ -62,6 +73,15 @@ def simulate_images(testnum, detim, ra, dec, do_xshift, do_rotation, supernova,
     galra = ra + 1.5e-5
     galdec = dec + 1.5e-5
 
+    if sim_lc is None:
```

Comment:
Why?  Document.  The continued use of `detim`, which I don't understand, is continuing to trip me up every time I try to read and understand a code block.